core/marketing_dogu.go

```diff
@@ -0,0 +1,222 @@
+//			"Title": "Dogu icon"
+//		},
+//		"BackgroundImage": {
+//			"ID": "18a5dc18-c202-40b2-9ee0-7f917ad82406",
+//			"Title": "Screenshot of the main page of the new dogu"
+//		},
+//		"Descriptions": [
+//			{
+//				"Description": "Ein neues Dogu",
+//				"LanguageCode": "de"
+//			},
+//			{
+//				"Description": "A new dogu",
+//				"LanguageCode": "en"
+//			}
+//		],
+//		"ReleaseNotes": "https://example.com/release-notes"
+//	}
+type MarketingDogu struct {
+	// ID contains the unique identifier of the dogu given by the CMS.
+	//
+	// Examples:
+	//   - 0191b1e2-350f-7ecf-a0a8-6f2a2f0d3607
+	//   - 34393ef9-a96d-4d1d-823e-0c17b10b762d
+	//
+	ID string
+	// Deprecated indicates if the dogu is deprecated.
+	//
+	Deprecated bool
+	// Namespace contains the dogu's namespace without the name.
+	//
+	// Name together with Namespace delimited by a single forward slash "/" builds the dogu's full qualified name.
+	//
+	// Examples:
+	//   - official
+	//   - premium
+	//
+	Namespace string
+	// Name contains the dogu's simple name without the namespace.
+	//
+	// Name together with Namespace delimited by a single forward slash "/" builds the dogu's full qualified name.
+	//
+	// Examples:
+	//   - redmine
+	//   - confluence
+	//   - newdogu
+	//
+	Name string
+	// DisplayName is the name of the dogu which is used in UI frontends to represent the dogu.
+	//
+	// Examples:
+	//  - Jenkins CI
+	//  - Backup & Restore
+	//  - My New Dogu
+	//
+	DisplayName string
+	// Provider contains information about the provider of the dogu.
+	//
+	// Example:
+	// 	{
+	//		"Slug": "provider",
+	//		"Name": "Provider",
+	//		"Logo": {
+	//			"ID": "34393ef9-a96d-4d1d-823e-0c17b10b762d",
+	//			"Title": "Provider Logo"
+	//		}
+	//	}
+	//
+	Provider []Provider
+	// Logo contains information about the logo or icon of the dogu.
+	//
+	// Example:
+	// 	{
+	//		"ID": "34393ef9-a96d-4d1d-823e-0c17b10b762d",
+	//		"Title": "Dogu Logo"
+	//	}
+	//
+	Logo Image
+	// BackgroundImage contains information about the background image of the dogu.
+	//
+	// Example:
+	//	{
+	//		"ID": "18a5dc18-c202-40b2-9ee0-7f917ad82406",
+	//		"Title": "Screenshot of the main page of the new dogu"
+	//	}
+	//
+	BackgroundImage Image
+	// Descriptions contains a short explanation, what the dogu does in different languages.
+	//
+	// Example:
+	// 	{
+	//		"Description": "Ein neues Dogu",
+	//		"LanguageCode": "de"
+	//	},
+	//	{
+	//		"Description": "A new dogu",
+	//		"LanguageCode": "en"
+	//	}
+	Descriptions []Translations
+	// ReleaseNotes contains an URL to the release notes of the dogu.
+	//
+	// Example:
+	//	- https://example.com/release-notes
+	ReleaseNotes string
+}
+
+// Provider describes properties of the provider of the dogu which can be used to represent the provider in UI frontends.
+//
+// Example:
+//
+//	{
+//		"Slug": "provider",
+//		"Name": "Provider",
+//		"Logo": {
+//			"ID": "34393ef9-a96d-4d1d-823e-0c17b10b762d",
+//			"Title": "Provider Logo"
+//		}
+//	}
```

Comment:
This looks nice and I feel it bears repeating the same example data throughout the nested field structure. Below you show Cloudogu as provider and we should continue to do so.

```suggestion
// Example:
//
//	{
//		"Slug": "cloudogu-provider-part",
//		"Name": "Cloudogu GmbH",
//		"Logo": {
//			"ID": "34393ef9-a96d-4d1d-823e-0c17b10b762d",
//			"Title": "Cloudogu Logo"
//		}
//	}
```